tg_bot.py

```diff
@@ -40,9 +41,15 @@ def display_tweet(self, context):
         print('starting display parameter')
         job = context.job
         global KILL_SIGNAL
+        previous_date =  datetime.now() + timedelta(days=-1)
         tid = pd.read_csv(self.dirpath + 'tid_store.csv')
-        for index, tid_store in tid.iterrows():
-            
+        tid["Date_time"]= pd.to_datetime(tid["Date_time"])
+        
+        for index, tid_store in tid[::-1].iterrows():
+            scrape_date = tid_store['Date_time']
+            if scrape_date<=previous_date:
```

Comment:
What is there is no previous date to pick? 

That is, imagine a scenarios where the last date we have is just 15/05 and you are requesting for 14/05 which doesn't exist in the tid store?

Reply:
it will still send the data right? `previous_date=datetime.now()-timedelta(days=1)` so regardless of `previous_date` existing in the file it will send

Reply:
Would it? And what if previous date is not in the tid store?

Reply:
Exactly as I said before, it will just stop sending `scrape_date<=previous_date` till this condition is satisfied. If previous date is not there it will continue sending the tweets till there is a previous date. `previous_date=datetime.now()-timedelta(days=1)`